src/site/content/en/learn/design/interaction/index.md

```diff
@@ -0,0 +1,275 @@
+authors:
+  - adactio
+date: 2021-12-23
+---
+
+
+Small screen devices like mobile phones often have touchscreens. Large screen devices like laptops and desktop computers often come with hardware like a mouse or a trackpad. It’s tempting to equate small screens with touch and large screens with pointers.
+
+But the reality is more nuanced. Some laptops have touch screen capabilities. Users can interact with either a touchscreen or a trackpad or both. Likewise it’s possible to use an external keyboard or mouse with a touchscreen device like a tablet.
+
+Instead of trying to infer the input mechanism from screen size, use media features in CSS.
+
+## Pointer
+
+You can test for three possible values with the `pointer` media feature: `none`, `coarse`, and `fine`.
+
+If a browser reports a `pointer` value of `none` the user might be using a keyboard to interact with your website.
+
+If a browser reports a `pointer` value of `coarse` it means the primary input mechanism isn’t very accurate. A finger on a touchscreen is a coarse pointer.
+
+If a browser reports a `pointer` value of `fine` it means the primary input mechanism is capable of fine-grained control. A mouse or stylus is a fine pointer.
+
+You can adjust the size of your interface elements to suit the `pointer` value. Try visiting [this website](https://gui-challenges.web.app/settings/dist/) in different kinds of devices to see how the interface adapts.
+
+In this example, buttons are made larger for coarse pointers:
+
+```css
+button {
+  padding: 0.5em 1em;
+}
+@media (pointer: coarse) {
+  button {
+    padding: 1em 2em;
+  }
+}
+```
+
+It’s possible to also make elements smaller for fine pointers but be careful about doing this:
+
+{% Compare 'worse' %}
+```css
+@media (pointer: fine) {
+  button {
+    padding: 0.25em 0.5em;
+  }
+}
+```
+{% endCompare %}
+
+Even if someone has a primary input mechanism capable of fine-grained control, think twice before reducing the size of interactive elements. [Fitts’s Law](https://en.wikipedia.org/wiki/Fitts's_law) still applies. A smaller target requires more concentration even with a fine pointer. A larger target area benefits everyone regardless of pointing device.
+
+## Any pointer
+
+The `pointer` media feature reports the fineness of the *primary* input mechanism. But many devices have more than one input mechanism. It’s possible that someone is interacting with your website using both a touchscreen and a mouse at the same time.
+
+The `any-pointer` differs from the `pointer` media feature in that it reports if any pointing device passes the test.
+
+An `any-pointer` value of “none” means that no pointing device is available.
+
+An `any-pointer` value of “coarse” means that at least one pointing device is not very accurate. But that might not be the primary input mechanism.
+
+An `any-pointer` value of “fine” means that at least one pointing device is capable of fine-grained control. But again, this might not be the primary input mechanism.
```

Comment:
```suggestion
An `any-pointer` value of `fine` means that at least one pointing device is capable of fine-grained control. But again, this might not be the primary input mechanism.
```